src/test/java/org/jabref/logic/net/URLUtilTest.java

```diff
@@ -87,4 +90,32 @@ void createUriShouldHandlePipeCharacter() {
         URI uri = URLUtil.createUri(input);
         assertEquals("http://example.com/test%7Cfile", uri.toString());
     }
+
+    @Test
+    void createShouldHandleRelativeURLs() throws Exception {
+        URL relativeUrl = URLUtil.create("www.example.com");
+        assertEquals("https://www.example.com", relativeUrl.toString());
+
+        URL noWwwUrl = URLUtil.create("example.com");
+        assertEquals("https://example.com", noWwwUrl.toString());
+    }
+
+    @Test
+    void createShouldHandleAbsoluteURLs() throws Exception {
+        URL absoluteHttpUrl = URLUtil.create("http://www.example.com");
+        assertEquals("http://www.example.com", absoluteHttpUrl.toString());
+
+        URL absoluteHttpsUrl = URLUtil.create("https://www.example.com");
+        assertEquals("https://www.example.com", absoluteHttpsUrl.toString());
+    }
```

Comment:
Convert to parameterized test

Reasons:

1. Not more than one assertion per test case 
2. Tests follow the same pattern as `createShouldHandleOtherProtocols`

You can use Java comments to seperate different test intentions in the CSVSource